addons/account_facturx/data/facturx_templates.xml

```diff
@@ -181,57 +176,54 @@
 
                     <!-- Taxes. -->
                     <ram:ApplicableHeaderTradeSettlement>
+                        <ram:InvoiceCurrencyCode t-esc="record.currency_id.name"/>
 
                         <!-- Bank account. -->
                         <ram:SpecifiedTradeSettlementPaymentMeans t-if="record.invoice_partner_bank_id.acc_type == 'iban'">
+                            <ram:TypeCode t-esc="'42'"/>
                             <ram:PayeePartyCreditorFinancialAccount>
                                 <ram:IBANID t-esc="record.invoice_partner_bank_id.sanitized_acc_number"/>
                             </ram:PayeePartyCreditorFinancialAccount>
                         </ram:SpecifiedTradeSettlementPaymentMeans>
 
                         <!-- Tax Summary. -->
                         <t t-foreach="tax_details" t-as="tax_vals">
-                            <t t-set="tax_line" t-value="tax_vals['line']"/>
                             <ram:ApplicableTradeTax>
                                 <ram:CalculatedAmount
-                                    t-att-currencyID="currency.name"
-                                    t-esc="format_monetary(tax_vals['tax_amount'], currency)"/>
+                                    t-esc="format_monetary(balance_multiplicator * tax_vals['tax_amount'], currency)"/>
```

Comment:
apply the multiplication python-side to avoid unconsistent values for poeple that didn't update the module